zgw/documenten-api/src/main/kotlin/com/ritense/portal/documentenapi/web/rest/DocumentContentResource.kt

```diff
@@ -58,7 +64,16 @@ class DocumentContentResource(
     }
 
     @PostMapping(value = ["/document/content"], consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
-    suspend fun uploadStreaming(@RequestPart("file") file: FilePart): ResponseEntity<Document> {
+    suspend fun uploadStreaming(@RequestPart("file") file: FilePart): ResponseEntity<Any> {
+        if (documentenApiVirusScanConfig.enabled) {
+            // virus scan is enabled
```

Comment:
Dit voegt niet veel meer toe dan de if die erboven staat

Reply:
maar je wilt toch weten of de virusscan gebruik moet worden?

Reply:
Als je de `virusScanService` nullable hebt gemaakt weet je dat ook. Als hij bestaat gebruik je hem, en anders niet.